src/envo/modules/entity_attribute_location.csv

```diff
@@ -20,15 +20,11 @@ ENVO:09000018,,PATO:0000033,concentration,CHEBI:25555,nitrogen atom,ENVO:0000200
 ENVO:09200006,,PATO:0000033,concentration,CHEBI:28938,ammonium,ENVO:00001998,soil
 ENVO:09200007,,PATO:0000033,concentration,CHEBI:17632,nitrate,ENVO:00001998,soil
 ENVO:09000019,,PATO:0000033,concentration,CHEBI:17996,chloride,ENVO:00002006,water
-# ENVO:09000020,,PATO:0000033,concentration,CHEBI:17632,nitrate,ENVO:01001004,groundwater # removed (equiv to: ENVO:09200020) before release, ID can be reused.
-# ENVO:09000021,,PATO:0000033,concentration,CHEBI:15379,oxygen,ENVO:00002006,water # removed (equiv to: ENVO_3100011) before release, ID can be reused.
```

Comment:
Commented rows are illegal in dosdp tools, see https://github.com/INCATools/dosdp-tools/issues/451